acceptance-tests/src/test/resources/features/UKDrivingLicence/English/Common/DLFullJourneyCommon.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Prove Your Identity Full Journey Common
```

Comment:
I do not believe we actually need these tests there just covering the same functionality as the tests above